test/unit/domain/oracle/oracle.test.js

```diff
@@ -40,33 +40,44 @@ const partyIdTypeResponse = {
   partyIdTypeId: 1,
   name: 'MSISDN',
   description: 'A MSISDN (Mobile Station International Subscriber Directory Number, that is, the phone number)',
-  isActive: true,
+  isActive: 1,
   createdDate: '2019-05-24 08:52:19'
 }
 
 const partyIdTypeResponseIBAN = {
   partyIdTypeId: 2,
   name: 'IBAN',
   description: 'An IBAN',
-  isActive: true,
+  isActive: 1,
```

Comment:
Why is this changing from a boolean to an int here? That perhaps points to unwanted API changes

Reply:
because the mysql database returns int's for boolean fields.
these variables passed through mocked implementation. 
they weren't being representative of what the db would actually return

Reply:
ok no worries, that just triggered my spidey sense...